apps/docs/src/components/document-renderer/components/index.tsx

```diff
@@ -64,6 +64,10 @@ export const components: MDXComponents = {
   tbody: Table.Body,
   td: Table.Cell,
   tr: Table.Row,
+  /** layout components */
+  Box: (props) => <Box {...props} />,
+  Flex: (props) => <Flex {...props} />,
+  Grid: (props) => <Grid {...props} />,
```

Comment:
Why not just this? Was there an issue?
```suggestion
  /** layout components */
  Box,
  Flex,
  Grid,
```

Reply:
no issue, I just saw that some of the other components were declared this way and figured it wouldnt hurt to be sure props are being passed